redbadger/website-honestly
# Full URL support for event featured images

![giphy 18](https://cloud.githubusercontent.com/assets/487468/22932519/1f1b0a20-f2c0-11e6-9768-18dd11502a1d.gif)

Refactoring event utils out of website-next, adding tests and functionality to handle full image URL and http to https conversion

### Motivation

- This fixes #344 . When full image URL is provided on Prismic, it will be allowed to be used as is. With one important detail - if the URL is `http://` it will be converted to `https://`.

Update (15.2.2017): URL resolving logic has been moved to Badger Brain. Website Honestly will rely on Badger Brain to provide valid URL:s for event featured images.

### Test plan

- Open Prismic staging. Edit existing event or create a new one. Add featured image as a full URL. Re-build the site. Check that the image is correctly displayed.
- Make sure that old images are still correctly displayed.
- Add image URL as `http://`. Check that the URL is converted to `https://` on the site.

The only way to update the site on preview app is to re-trigger CircleCI build for this branch. Same goes for the staging.

### Pre-merge checklist
